_layouts/cask.html

```diff
@@ -83,6 +83,26 @@
     </tr>
 </table>
 {%- endif %}
+{%- if c.variations.size > 0 -%}
```

Comment:
```suggestion
{%- if c.variations.size > 1 -%}
```

maybe because a single variation is not very interesting/useful.